src/test/java/org/jabref/migrations/SpecialFieldsToSeparateFieldsTest.java

```diff
@@ -20,7 +20,7 @@ class SpecialFieldsToSeparateFieldsTest {
     @ParameterizedTest
     @MethodSource("provideKeywordFieldPairs")
     public void migrateToCorrectField(SpecialField field, String fieldInKeyword, BibEntry expected) {
-        BibEntry entry = new BibEntry().withField(StandardField.KEYWORDS, fieldInKeyword);
+        BibEntry entry = new BibEntry().withField(field, fieldInKeyword);
```

Comment:
The intention of this test class is to check migration of special field values in the `keywords` field. Thus, please to not alter this test.

In the past, JabRef offered storing special fields in keywords (see https://github.com/JabRef/jabref/blob/main/docs/decisions/0019-implement-special-fields-as-separate-fields.md#special-fields-as-keywords). We decided to move away from that.